tetris.js

```diff
@@ -229,10 +247,15 @@ module.exports = {
 		return newArr;
 	},
 	
-	buildTetromino: function(stringOfCoordinates){
-		let math = stringOfCoordinates.split("\n")
-		let arrayOfCoordinates = []
+	getCoordinates: function(stringOfCoordinates){
+		let math
 		let quantityElem = 0;
+		let arrayOfCoordinates = []
```

Comment:
You created this empty array and are returning it on line 282. But you are not updating it anywhere - looks like a bug

Reply:
I work with this array

Reply:
Ok